src/codeu/chat/client/commandline/Chat.java

```diff
@@ -305,12 +320,28 @@ public void invoke(List<String> args) {
           if (conversation == null) {
             System.out.println("ERROR: Failed to create new conversation");
           } else {
+            updateInterests(conversation.conversation);
             panels.push(createConversationPanel(conversation));
           }
         } else {
           System.out.println("ERROR: Missing <title>");
         }
       }
+
+      private void updateInterests(ConversationHeader conversation){
```

Comment:
Add a comment explaining what the feature accomplishes, so coders don't have to try to read the code to figure out the purpose.